cmd/amplifier/main.go

```diff
@@ -52,6 +56,8 @@ func parseFlags() {
 	flag.StringVarP(&clientSecret, "clientsecret", "c", defaultClientSecret, "github app clientsecret (default '"+defaultClientSecret+"')")
 	flag.StringVarP(&kafkaURL, "kafkaURL", "k", kafkaDefaultURL, "kafka URL (default '"+kafkaDefaultURL+"')")
 	flag.StringVarP(&influxURL, "influxURL", "", influxDefaultURL, "InfluxDB URL (default '"+influxDefaultURL+"')")
+	flag.StringVarP(&dockerURL, "dockerURL", "", dockerDefaultURL, "Docker URL (default '"+dockerDefaultURL+"')")
```

Comment:
Those should also be updated, but we don't have to hold up the PR for you. It can be done separately later.